crowdsec-docs/docs/bouncers/aws-waf.mdx

```diff
@@ -0,0 +1,203 @@
+---
+id: aws_waf
+title: AWS WAF Bouncer
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p align="center">
+<img src={useBaseUrl('/img/TODO_FIXME.png')} alt="CrowdSec" title="CrowdSec" width="300" height="300" />
+</p>
+
+<p align="center">
+&#x1F4DA; <a href="#installation/">Documentation</a>
+&#x1F4A0; <a href="https://hub.crowdsec.net">Hub</a>
+&#128172; <a href="https://discourse.crowdsec.net">Discourse </a>
+</p>
+
+## Overview
+
+The `crowdsec-awf-waf-bouncer` automatically adds rules to an existing AWS WAF ACL and manages IPSets content to apply decisions taken by crowdsec.
+
+This allows to protect the following AWS ressources with crowdsec:
+ - AWS REST API Gateway
+ - Cloudfront distribution
+ - AWS Application LoadBalancer
+ - AWS AppSync GraphQL API
+
+As the bouncer does not manage your WAF ACLs, you will need to have existing ACLs already associated with your AWS ressources for the bouncer to work properly.
```

Comment:
maybe add early that it supports both captcha, ban, and geolocalistion decisions. Which scope does it picks up for country ? `Country` ?